packages/katello/katello/backup.rb

```diff
@@ -87,7 +87,8 @@ def configure_configs
         '/opt/puppetlabs/puppet/cache/foreman_cache_data',
         '/opt/puppetlabs/puppet/ssl/',
         '/var/lib/puppet/foreman_cache_data',
-        '/var/lib/puppet/ssl'
+        '/var/lib/puppet/ssl',
+        '/usr/share/foreman-proxy/.ssh/'
```

Comment:
This does make me wonder: why is it writing in ```/usr```? FHS says this shouldn't happen and as an admin I'd be surprised about this behavior. Why doesn't it live in ```/var/lib/foreman-proxy```?

Reply:
foreman-proxy $HOME is /usr/share/foreman-proxy

Reply:
It sounds like something would should fix, possibly via a symlink.